Header header
depth_image_extractor/BoundingBox2D[] bboxes